cluster/gce/gci/configure-helper.sh

```diff
@@ -2207,6 +2207,8 @@ function start-kube-controller-manager {
   fi
   if [[ -n "${RUN_CONTROLLERS:-}" ]]; then
     params+=("--controllers=${RUN_CONTROLLERS}")
+  elif [[ "${CLOUD_PROVIDER_FLAG:-external}" == "external" ]]; then
```

Comment:
this interaction between RUN_CONTROLLERS and CLOUD_PROVIDER_FLAG isn't obvious... should we always tack on -nodeipam if `"${CLOUD_PROVIDER_FLAG:-external}" == "external"`, whether RUN_CONTROLLERS is set or not?

Reply:
I think the default answer is yes. 

There is a special case if you are migrating from KCM only to KCM + CCM, where the answer might be no. If you want to migrate one controller at a time (which the code supports), there would be a period where you disable nodeipam on the CCM and don't disable it on the KCM. However that is not the normal behavior. 

Generally if you are running the CCM (CLOUD_PROVIDER_FLAG == "external" and your running on GCP) then you want to disable the nodeipam controller on the KCM.